### PR TITLE
Clarify what happens when a duplicate subscription is sent, remove CLOSED on duplicate subscription

### PR DESCRIPTION
**Pull request by staab**
Fixes https://github.com/nostr-protocol/nips/issues/1332

### CODE REVIEW

**Review by mikedilger — APPROVED** [review]

**Comment by bezysoftware** [comment]
LGTM!

**Comment by bezysoftware** [comment]
Ping @fiatjaf

File: 01.md [hunk 1]
```diff
@@ -169,7 +169,6 @@ This NIP defines no rules for how `NOTICE` messages should be sent or treated.
   * `["OK", "b1a649ebe8...", false, "pow: difficulty 26 is less than 30"]`
   * `["OK", "b1a649ebe8...", false, "error: could not connect to the database"]`
 - `CLOSED` messages MUST be sent in response to a `REQ` when the relay refuses to fulfill it. It can also be sent when a relay decides to kill a subscription on its side before a client has disconnected or sent a `CLOSE`. This message uses the same pattern of `OK` messages with the machine-readable prefix and human-readable message. Some examples:
-  * `["CLOSED", "sub1", "duplicate: sub1 already opened"]`
```
Comment (TheRebelOfBabylon) [inline 01.md:172]: Suggest we replace this example with one that may lead to more relays enforcing the max sub id length

* `["CLOSED", "sub1", "error: subscription id too large"]`

**Comment by mikedilger** [comment]
ACK

**Review by cameri — APPROVED** [review]

**Review by fiatjaf — APPROVED** [review]

**Pull request merged**